src/app/detail/ui/Sidebar.tsx

```diff
@@ -79,7 +79,7 @@ export const Sidebar = ({
     }
   }, [tasks, workflowStates])
 
-  const matches = useMediaQuery('(max-width:600px)')
+  const matches = useMediaQuery('(max-width:700px)')
   if (!tasks) return null
 
```

Comment:
The initial assumption for the medium screen width is 600px which was also shared in this issue and was decided accordingly - https://linear.app/copilotplatforms/issue/OUT-1/create-tokens

Changing this to 700px over here is going to change the overall responsive idea of the entire app.